app/assets/stylesheets/forms.css

```diff
@@ -1,3 +1,8 @@
 .help-text {
 	color: #999999;
+}
+
+.input-select .select-wrapper {
```

Comment:
I've never used the materialize library before so there might be a class that does this already.